test/integration/rtpbroadcast.js

```diff
@@ -0,0 +1,105 @@
+describe('Rtpbroadcast tests', function() {
+  var janusConnection;
+  var janusSession;
+  var rtpbroadcastPlugin;
+
+  var mountpointOptions = {
+    name: 'Delete me',
+    description: 'I must be deleted soon',
+    recorded: true,
+    streams: [{
+      "audiopt": 111,
+      "audiortpmap": "opus/48000/2",
+      "videopt": 100,
+      "videortpmap": "VP8/90000"
+    }]
+  };
+
+  function randomMountpointId() {
+    return Math.random().toString(36).substring(2, 12);
+  }
+
+  before(function(done) {
+    $('#mocha').append('<video id="video" autoplay></video>');
+
+    jQuery.getJSON('./config.json')
+      .then(function(config) {
+        var janus = new Janus.Client(config.url, config);
+        return janus.createConnection('client');
+      })
+      .then(function(connection) {
+        janusConnection = connection;
+        return connection.createSession();
+      })
+      .then(function(session) {
+        janusSession = session;
+        done();
+      });
+  });
+
+  after(function(done) {
+    janusSession.destroy()
+      .then(function() {
+        return janusConnection.close();
+      })
+      .then(done);
+  });
+
+  beforeEach(function(done) {
+    janusSession.attachPlugin(Janus.RtpbroadcastPlugin.NAME)
+      .then(function(plugin) {
+        rtpbroadcastPlugin = plugin;
+        done();
+      });
+  });
+
+  afterEach(function(done) {
+    rtpbroadcastPlugin.detach().then(done);
+  });
+
+  it('creates, lists and destroys', function(done) {
+    var mountpointId = randomMountpointId();
+    rtpbroadcastPlugin.create(mountpointId, mountpointOptions)
+      .then(function(response) {
+        assert.equal(response['created'], mountpointOptions['name']);
+        return rtpbroadcastPlugin.list(mountpointId);
+      })
+      .then(function(list) {
+        assert.equal(list.length, 1);
+        assert.equal(list[0]['id'], mountpointId);
+        return rtpbroadcastPlugin.destroy(mountpointId);
+      })
+      .then(function(response) {
+        assert.equal(response['plugindata']['data']['destroyed'], mountpointId);
+        done();
+      });
+  });
+
+  it('creates, watches, pauses, starts, stops and destroys', function(done) {
```

Comment:
maybe some assertions could be added for this test (`addstream` triggered, `PeerConnection` state,...)